JM/CF/Scripts/1_Core/CommunityFramework/CF_Cast.c

```diff
@@ -0,0 +1,13 @@
+class CF_Cast<Class U, Class V>
+{
+	static V Reinterpret(U src)
+	{
+		U src_arr[1];
+		src_arr[0] = src;
+		V dst_arr[1];
+
+		copyarray(dst_arr, src_arr);
+
+		return dst_arr[0];
+	}
+};
```

Comment:
New line at the end of files please.

Reply:
Weird, it has a new line for me on my PC. Git isn't detecting new file changes.

Reply:
I do not think git is to blame here, even the addition of an empty line would be a stageable change

Reply:
Only Github shows it like this ... when cloned there is a newline.